cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split2/SpeakerEmotionInferenceSuite.scala

```diff
@@ -0,0 +1,164 @@
+package com.microsoft.azure.synapse.ml.cognitive.split2
+
+import com.microsoft.azure.synapse.ml.cognitive.SpeakerEmotionInferenceFormat._
+import com.microsoft.azure.synapse.ml.cognitive._
+import com.microsoft.azure.synapse.ml.core.test.fuzzing.{TestObject, TransformerFuzzing}
+import org.apache.commons.io.FileUtils
+import org.apache.spark.ml.util.MLReadable
+import org.apache.spark.sql.DataFrame
+import spray.json._
+
+import java.io.File
+import java.nio.file.Files
+
+class SpeakerEmotionInferenceSuite
+  extends TransformerFuzzing[SpeakerEmotionInference] with CognitiveKey {
+
+  import spark.implicits._
+
+  lazy val saveDir: File = Files.createTempDirectory("AudioFiles-").toFile
+
+  override def afterAll(): Unit = {
+    if (saveDir.exists()) {
+      FileUtils.forceDelete(saveDir)
+    }
+    super.afterAll()
+  }
```

Comment:
nit: i think theres already a temp dir availible in base class of tests

Reply:
Swapped a few other deriving classes to use the base class' temp dir as well. Let me know if you want me to revert.